### ISSUE TITLE
Handle assets too

### ISSUE THREAD

**Issue by sandstrom** [treyhunner/ember-deploy-ssh-index#14]
Would be neat if this could handle assets too? I.e. if both the index and assets are deployed to via SSH.

**Comment by treyhunner** [comment]
I agree.  I also have a need for deploying all assets over SSH in some of my projects.

I named the project this way so there could eventually be a separate ember-deploy-ssh project.  I may start that new project myself, but I want to finish the tests branch first before I copy/fork the code.

**Comment by sandstrom** [comment]
Sounds awesome! It would indeed be very useful! :boat:

**Comment by treyhunner** [comment]
It looks like made @eddflrs [ember-cli-deploy-ssh](https://github.com/eddflrs/ember-cli-deploy-ssh) for handling assets.
